ace-window-posframe.el

```diff
@@ -0,0 +1,37 @@
+(defvar aw--posframe-frames '())
+
+(defun aw--lead-overlay-posframe (path leaf)
+  (let* ((wnd (cdr leaf))
+         (str (apply #'string path))
+         (bufname (format "*aw-posframe-buffer-%s*" (gensym))))
+    (with-selected-window wnd
+      (push bufname aw--posframe-frames)
+      (posframe-show bufname
+                     :string str
+                     :poshandler 'posframe-poshandler-window-center
+                     :font (face-font 'aw-leading-char-face)
+                     :foreground-color (face-foreground 'aw-leading-char-face)
+                     :background-color (face-background 'aw-leading-char-face)))))
+
+(defun aw--remove-leading-chars-posframe ()
+  (map nil #'posframe-delete aw--posframe-frames))
```

Comment:
Sorry for my ignorance, but while trying this I got an error about map not being defined.  I don't know which package should provide this (I tried to install and load a package called "map", but it didn't work).  A simpler

    (mapcar #'posframe-delete aw--posframe-frames)

seems to work. (I only did some quick testing).

Another thing is that using your fork, posframe is not listed as dependency (or at least straight.el doesn't think it is -- I am not yet aware of how dependencies are declared and consumed.)

Anyway, thanks for this feature, is awesome :)